### PR TITLE
Feat: UI gradient emotes

### PR DESCRIPTION
**Pull request by JuanSalice**
## What does this PR change?

This PR adds the custom gradient elements to the emotes wheel selector, which is now animated.

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/UI-Gradient-Emotes
2. Open the emotes wheel.
3. Put the mouse over an element of the wheel.
4. Verify the selector is visible and animated.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: unity-renderer/Assets/Rendering/UI/Rainbow/UIRadialRainbow.cs
Comment (sandrade-dcl) [inline unity-renderer/Assets/Rendering/UI/Rainbow/UIRadialRainbow.cs:8]: In order to follow our code conventions, lets try to avoid the use of `_` in the variables:

```suggestion
    Material targetMat;
```

File: unity-renderer/Assets/Rendering/UI/Rainbow/UIRadialRainbow.cs
Comment (sandrade-dcl) [inline unity-renderer/Assets/Rendering/UI/Rainbow/UIRadialRainbow.cs:19]: In order to have a mode readable code, lets put an space here, between the variables block and the header of the next function:

```suggestion
    public float rotation;
    
    void Start()
```

File: unity-renderer/Assets/Rendering/UI/Rainbow/UIRainbowController.cs
Comment (sandrade-dcl) [inline unity-renderer/Assets/Rendering/UI/Rainbow/UIRainbowController.cs:8]: In order to follow our code conventions, lets try to avoid the use of `_` in the variables:

```suggestion
    Material targetMat;
```

**Review by sandrade-dcl — APPROVED** [review]
I only left you some little suggestions related to our code conventions to apply, please take a quick look to them. Anyway I have left my approval in order to not leave the PR blocked. 

Lets wait for Romi's visual approval and we can merge it! 👍

**Comment by RominaMarchetti** [comment]
Looking good, Juan!
This comment is not a blocker to merge the branch: can you make the white outline the same on all sides of the shape? You should thicken the sides.
Screen Shot 2022-05-24 at 11 30 46

**Review by RominaMarchetti — APPROVED** [review]

**Pull request merged**
